src/components/custom-media-library/HeaderDeleteButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { useIntl } from "react-intl";
+import { FieldGroupItem } from "../dp-components/FieldGroup";
+import { CSSProperties } from "react";
+
+// TODO: consider moving these styles to the class col--auto-size in the Style Guide
+// The class name should start with `col-` to share padding values.
+const deleteButtonFieldGroupItemStyles: CSSProperties = { width: "unset" };
+const deleteButtonStyles: CSSProperties = { padding: "11px", fontSize: "1rem" };
```

Comment:
@GusBaamonde these are my customizations for the delete button.

Do you think that is there a way to move them to the style guide or reusing current styles here?